ethereum/rpc/backend/backend.go

```diff
@@ -204,7 +204,7 @@ func (e *EVMBackend) EthBlockFromTendermint(
 		}
 	}
 
-	blockBloomResp, err := e.queryClient.BlockBloom(types.ContextWithHeight(block.Height), &evmtypes.QueryBlockBloomRequest{})
+	blockBloomResp, err := e.queryClient.BlockBloom(types.ContextWithHeight(block.Height), &evmtypes.QueryBlockBloomRequest{Height: block.Height})
```

Comment:
Note :

types.ContextWithHeight(block.Height) => Append height in the header to tell tendermint to use storage at height but the header will be removed in deeper layer